src/test/java/formflow/library/utilities/AbstractMockMvcTest.java

```diff
@@ -331,6 +331,15 @@ protected void postExpectingFailureAndAssertErrorsDisplayForThatInput(String pag
     assertInputHasErrors(pageName, inputName, errorMessages);
   }
 
+  protected void postExpectingFailureAndAssertInputErrorMessages(String pageName,
+      Map<String, String> inputParams,
+      Map<String, List<String>> expectedErrorMessages) throws Exception {
+//    postExpectingFailure(pageName, inputParams);
```

Comment:
How are there error messages without this post?

Reply:
I think this test really operates via the `assertInputHasErrors` call: https://github.com/codeforamerica/form-flow/pull/134/files#diff-63e6c79523ceedc33846c5a80d3fb2613b1124f70e5374c7338e4ae3c19ef8eaR398

Reply:
If the change to the assert function below makes sense, I think we can uncomment this line.